### PR TITLE
[MINOR] Update Readme for v1.1.0

### PR DESCRIPTION
**Pull request by pan3793**
pan3793 [Closes #426](https://github.com/yaooqinn/kyuubi/pull/426)    Target Issue Test Plan [&#10088;?&#10089;](https://pullrequestbadge.com/?utm_medium=github&utm_source=yaooqinn&utm_campaign=badge_info)

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ You can find the specific version of Kyuubi documentation as listed below.
 
 - [master/latest](https://kyuubi.readthedocs.io/en/latest/)
 - [stable](https://kyuubi.readthedocs.io/en/stable/)
+- [v1.1.0](https://kyuubi.readthedocs.io/en/v1.1.0/)
 - [v1.0.2](https://kyuubi.readthedocs.io/en/v1.0.2/)
```
Comment (yaooqinn) [inline README.md:43]: we need v1.0.3 too

Comment (pan3793) [reply README.md:44]: updated

**Review by yaooqinn — APPROVED** [review]
